boards.txt

```diff
@@ -26494,6 +26494,233 @@ fm-devkit.menu.EraseFlash.none.upload.erase_cmd=
 fm-devkit.menu.EraseFlash.all=Enabled
 fm-devkit.menu.EraseFlash.all.upload.erase_cmd=-e
 
+##############################################################
+### Fri3d Badge 2024 (ESP32-S3-WROOM-1)
+
+fri3d_2024_esp32s3.name=Fri3d Badge 2024 (ESP32-S3-WROOM-1)
+fri3d_2024_esp32s3.vid.0=0x303a
+fri3d_2024_esp32s3.pid.0=0x1001
+
+fri3d_2024_esp32s3.bootloader.tool=esptool_py
+fri3d_2024_esp32s3.bootloader.tool.default=esptool_py
+
+fri3d_2024_esp32s3.upload.tool=esptool_py
+fri3d_2024_esp32s3.upload.tool.default=esptool_py
+fri3d_2024_esp32s3.upload.tool.network=esp_ota
+
+fri3d_2024_esp32s3.upload.maximum_size=1310720
+fri3d_2024_esp32s3.upload.maximum_data_size=327680
+fri3d_2024_esp32s3.upload.flags=
+fri3d_2024_esp32s3.upload.extra_flags=
+fri3d_2024_esp32s3.upload.use_1200bps_touch=false
+fri3d_2024_esp32s3.upload.wait_for_upload_port=false
+
+fri3d_2024_esp32s3.serial.disableDTR=false
+fri3d_2024_esp32s3.serial.disableRTS=false
+
+fri3d_2024_esp32s3.build.tarch=xtensa
+fri3d_2024_esp32s3.build.bootloader_addr=0x0
+fri3d_2024_esp32s3.build.target=esp32s3
+fri3d_2024_esp32s3.build.mcu=esp32s3
+fri3d_2024_esp32s3.build.core=esp32
+fri3d_2024_esp32s3.build.variant=fri3d_2024_esp32s3
+fri3d_2024_esp32s3.build.board=FRI3D-BADGE-2024_ESP32S3-WROOM-1
```

Comment:
One more thing, - (dash) is not valid character for build.board name. Can you please fix it as suggestion?
```suggestion
fri3d_2024_esp32s3.build.board=FRI3D_BADGE_2024_ESP32S3_WROOM_1
```
Or maybe do it simpler by removing the WROOM_1, the esp32s3 can remain :)

Reply:
Thanks a lot for checking this PR and the suggestions!
I change this and then also still moved the order of options for better defaults. But I think all should be good now